### ISSUE TITLE
Change the color of the coming soon Button

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#252]
Navigate here: https://prime-deals-staging.vercel.app/initiate

Change the color of the coming soon button to:
#98979B

Result:
![Initiate-a-Deal-Prime-Deals (1)](https://user-images.githubusercontent.com/24368665/156803675-223065e7-1ea9-4722-a97c-62a8fb6c8aae.png)

Expected:
![Group 356](https://user-images.githubusercontent.com/24368665/156803669-fd957edd-2f6a-4b7f-bfb5-9c488911af0e.png)

**Comment by hiaux0** [comment]
@BartuCulha 

I checked the current design system [1], that specify for
`Neutral2` -> `#9398a8`
(vs. what the current Mocks use `#98979B`)

You or me check back with Jaap?

[1]: https://www.figma.com/file/CU4dt2fp3FmKZ0K1As0DJH/PrimeDAO-Design-System?node-id=340%3A338

**Comment by hiaux0** [comment]
Also for the future I'd like to kindly request to also include the name of the color (in this case "Neutral2").
One can find the name of the color in the sidebar in Figma

![image](https://user-images.githubusercontent.com/30693990/157023174-79784a27-121a-4829-a257-ccb09be0be8e.png)

**Comment by BartuCulha** [comment]
Yep you are right @hiaux0 . Sorry I did not know you were also using names. Jaap actually used Neutral2 during his feedback and I searched Figma to fetch the hex code  :D

**Comment by hiaux0** [comment]
@BartuCulha 
I'll add a comment to Figma to change the "old" color to the "new" one then?

And this issue is thus obsolete?

**Comment by BartuCulha** [comment]
Hmm the name of the color should be updated but also the UI. We use a different shade of gray

**Comment by BartuCulha** [comment]
Yes we should change the button color to #98979B

**Comment by hiaux0** [comment]
To be clear, we should change 
`Neutral2` -> `#98979B`?

**Comment by BartuCulha** [comment]
I think we should but lets wait for Jaap's answer

**Comment by hiaux0** [comment]
Jaap Telegram
> but [#9398a8]() is the right color

(code wise, keep as is)
